DependencyInjection/SonataAdminExtension.php

```diff
@@ -212,7 +212,9 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->setParameter('sonata.admin.configuration.show.mosaic.button', $config['show_mosaic_button']);
 
-        $this->configureClassesToCompile();
+        if (\PHP_VERSION_ID < 70000) {
```

Comment:
Please wait for decision here:
https://github.com/sonata-project/SonataCoreBundle/pull/409/files#r122891158